### ISSUE TITLE
False positive for jsx-indent in plain JS

### ISSUE THREAD

**Issue by amneacsu** [yannickcr/eslint-plugin-react#2565]
After upgrade from 17.8.0 to 17.8.2, `jsx-indent` now complains for plain strings in plain JS files. I was expecting `eslint-plugin-react` to ignore this file, as it's not importing React or using JSX.

```js
// somefile.js
const foo = {
  bar: 'baz1\nbaz2'
       ^ Expected indentation of 4 space characters but found 0
};
```

**Comment by amneacsu** [comment]
Fixed in 17.8.3.

**Comment by ljharb** [comment]
Duplicate of #2563.
